### DISCUSSION TITLE
How to prevent facial recognition

### DISCUSSION THREAD

**Discussion by pippo73** [immich-app/immich#14079] [Q&A]
Hi all, 
I have a question.
I did some photos at a big event with a lot of people. I've uploaded this photos to the server and now my people page is croud of unknown persons.
Is there a way to prevent the server to scan all the photos and add this persons?

**Comment by mertalev** [comment]
You can go to the [Facial Recognition Settings](http://my.immich.app/admin/system-settings?isOpen=machine-learning+facial-recognition) and increase "Minimum recognized faces" to something higher.

**Reply by pippo73** [reply]
If I do this, it will affect all the photos on my server. I would like just to skip it on an album (or else how to identify this photos) Anyway I don't know any of them ;-D
